.github/workflows/publish-prod-images.yml

```diff
@@ -72,10 +72,10 @@ jobs:
           uses: docker/build-push-action@v3
           continue-on-error: true
```

Comment:
> Also there is an issue with pushing of SigClient image, the existing [job fails at pushing of SigClient image](https://github.com/siglens/siglens/actions/runs/10744047913/job/29800103071#step:4:227) (not sure why it says the job is successful)

It says the job is successful because of this line:
```
continue-on-error: true
```

We probably want to keep this line, but maybe we could have another step/job that checks if any of the steps/jobs failed, and so we mark the whole workflow as failed if anything failed. Not sure if there's a better way to do this, and I'm not sure of the priority of doing this